lib/test_dispatch/form.ex

```diff
@@ -243,4 +243,16 @@ defmodule TestDispatch.Form do
   end
 
   defp deep_merge(_original, preceding), do: preceding
+
+  defp get_in_map(nil, [_], default_value), do: default_value
+  defp get_in_map(map, [key], default_value), do: Map.get(map, key, default_value)
+
+  defp get_in_map(nil, [_ | _], default_value), do: default_value
+
+  defp get_in_map(map, [key | keys], default_value) when is_map(map) do
+    value = Map.get(map, key)
+    get_in_map(value, keys, default_value)
+  end
+
+  defp get_in_map(_, _, default_value), do: default_value
```

Comment:
So what i'm trying to do here is returning a default_value when the final key wasn't found in the map. If the key was found however that value is returned. This means that if the value in the map for the last key is nil or false that won't be overwritten by the form_value at  https://github.com/DefactoSoftware/test_dispatch/pull/113/files#diff-9120c778da21edf9d7018c894df79e554720214682485a1e9fc18033e3ce1b42R92